xbmc/powermanagement/PowerManager.cpp

```diff
@@ -62,6 +62,8 @@ using namespace ANNOUNCEMENT;
 CPowerManager g_powerManager;
 
 CPowerManager::CPowerManager()
+  : m_lastFileItem(new CFileItem)
```

Comment:
keep the smart pointer empty, if now used

Reply:
thanks for the keyword "smart pointer"